packages/blocks/src/__internal__/utils/hotkey.ts

```diff
@@ -27,6 +27,19 @@ class HotkeyManager {
   enableHotkey() {
     this._setScope('page');
   }
+
+  /**
+   * Create a context to shielding against global hotkey
+   */
+  async withHotkeyShield(fn: () => void | Promise<unknown>) {
```

Comment:
Seems that `switchContext` can be more expressive about the intention in comment. The concept of `Shield` looks rare in related API design.

Reply:
Now updated to `withDisabledHotkey`